APM-70: PRUNE with output variable (#760)
arangodb/docs: 3.9/release-notes-new-features39.md

AQL
---

### Prune Variable 

Added option to store PRUNE expression as a variable: 

> Now, a PRUNE condition can be stored in a variable and be used later alongside other
> AQL keywords as their conditions.

Reviewer: ```suggestion
Now, a PRUNE condition can be stored in a variable and be used later in the query without
having to repeat the PRUNE condition.
```
